test/typescript/apis/project/typical.ts.OFF

```diff
@@ -0,0 +1,46 @@
+// verifies typical legacy import patterns
+// formerly through the legacy facade in @sap/cds/apis/...
+// now through official cds facade
```

Comment:
@daogrady I have decided not to delete the legacy test, but to keep it (disabled), so that we can check each import for its new equivalent.